cfgov/v1/tests/test_filterable_list.py

```diff
@@ -47,3 +45,26 @@ def get_site(self):
                 return None
 
         self.assertFalse(MockPageInDefaultSite().filterable_pages().exists())
+
+    def test_do_not_index_is_false_by_default(self):
+        assert self.mixin.do_not_index == False
```

Comment:
For consistency, would it make sense to use `self.assertFalse()` here and in the methods below?

Reply:
@willbarton I was just following convention within this file when I added these tests, but I had the same thought.  It seems like we've got a few different patterns going on in our back-end tests that I'd like us to standardize on at some point, but I feel that's out of scope for this PR.